py-notebook/jupyter-widget/js/webpack.config.js

```diff
@@ -0,0 +1,78 @@
+var path = require('path');
+var version = require('./package.json').version;
+
+// Custom webpack rules are generally the same for all webpack bundles, hence
+// stored in a separate local variable.
+var rules = [
+    { test: /\.css$/, use: ['style-loader', 'css-loader']}
+]
+
+
+module.exports = (env, argv) => {
+    var devtool = argv.mode === 'development' ? 'source-map' : false;
+    return [
+        {// Notebook extension
+        //
+        // This bundle only contains the part of the JavaScript that is run on
+        // load of the notebook. This section generally only performs
+        // some configuration for requirejs, and provides the legacy
+        // "load_ipython_extension" function which is required for any notebook
+        // extension.
+        //
+            entry: './lib/extension.js',
+            output: {
+                filename: 'extension.js',
+                path: path.resolve(__dirname, '..', 'neo_confusion_jupyter', 'nbextension'),
+                libraryTarget: 'amd',
+                publicPath: '' // publicPath is set in extension.js
+            },
+            devtool
+        },
+        {// Bundle for the notebook containing the custom widget views and models
+        //
+        // This bundle contains the implementation for the custom widget views and
+        // custom widget.
+        // It must be an amd module
+        //
+            entry: './lib/index.js',
+            output: {
+                filename: 'index.js',
+                path: path.resolve(__dirname, '..', 'neo_confusion_jupyter', 'nbextension'),
+                libraryTarget: 'amd',
+                publicPath: '',
+            },
+            devtool,
+            module: {
+                rules: rules
+            },
+            externals: ['@jupyter-widgets/base']
+        },
+        {// Embeddable neo-confusion-jupyter bundle
```

Comment:
I'm guessing this is related to the answer to my question above (about why neo-confusion-jupyter needs to be a package). How would people use this version? Is there a great advantage over the pip install approach?